src/coreclr/tools/aot/ILCompiler.TypeSystem.Tests/ILCompiler.TypeSystem.Tests.csproj

```diff
@@ -14,6 +14,7 @@
     <Platforms>AnyCPU;x64</Platforms>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants>READYTORUN;$(DefineConstants)</DefineConstants>
```

Comment:
We should not have any PInvoke stubs that do any actual marshaling in CoreLib anymore. It should be possible to clean up these ifdefs by stripping the bit of marshaling generation from crossgen2 that it still has (ie only generate stubs that do not require any actual marshaling in crossgen2).

It does not have to be done in this PR, but it may be nice to add comment.

Reply:
I've added READYTORUN as a shortcut because without it I would need to include InteropStateManager.cs and that depends on a bunch of more files. It was convenient.

https://github.com/dotnet/runtime/blob/6e286dd75eccf607d2b17795bc23b6a82bf0d97d/src/coreclr/tools/Common/TypeSystem/Interop/IL/MarshalHelpers.cs#L16-L18

I'll put it in a comment.